src/fdk_rdf_parser/reference_data/extend_dataset.py

```diff
@@ -68,12 +68,15 @@ def extend_distributions(
                                 lic.prefLabel = ref_code.prefLabel
                         extended_licenses.append(lic)
                     dist.license = extended_licenses
+
+            print("media_types")
             if media_types is not None:
                 if dist.format is not None:
                     extended_mediatypes = []
                     for fmt in dist.format:
                         if fmt is not None:
                             ref_code = media_types.get(fmt.lower())
+                            print("ref_code", ref_code)
```

Comment:
Fjerne denne, anbefaler heller å bruke "debug" i f.eks. PyCharm for å se variabler osv.